### PR TITLE
Added `DownloadRepodataSlice`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #32 
Added slice to download `repodata.json` items from storage. If the index does not exist, an empty json is returned. Conda client failed installation if `NOT FOUND` is returned in such cases.

### CODE REVIEW

File: src/test/java/com/artipie/conda/http/DownloadRepodataSliceTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import com.artipie.http.rq.RqMethod;
+import org.hamcrest.MatcherAssert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test for {@link DownloadRepodataSlice}.
+ * @since 0.4
+ */
+class DownloadRepodataSliceTest {
```
Comment (g4s8) [inline src/test/java/com/artipie/conda/http/DownloadRepodataSliceTest.java:23]: `DownloadRepodataSlice` appends `Content-Disposition` headers which could be checked in this test

File: src/main/java/com/artipie/conda/http/DownloadRepodataSlice.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                                content = CompletableFuture.completedFuture(
+                                    new Content.From("{}".getBytes(StandardCharsets.US_ASCII))
+                                );
+                            }
+                            return content;
+                        }
+                    ).thenApply(
+                        content -> new RsFull(
+                            RsStatus.OK,
+                            new Headers.From(new ContentFileName(key.toString())), content
```
Comment (g4s8) [inline src/main/java/com/artipie/conda/http/DownloadRepodataSlice.java:67]: `attachment` of `Content-Disposition` header should provide only file name, not full path, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition

So we need to extract file name from key (the last part after `/`)

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check my comments

**Comment by olenagerasimova** [comment]
@g4s8 thanks, corrected

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**